app/models/booking.rb

```diff
@@ -0,0 +1,7 @@
+class Booking < ApplicationRecord
+  validates :check_in, presence: true, uniqueness: true
+  validates :check_out, presence: true, uniqueness: true
+  has_many :bookings
+  belongs_to :user_id
```

Comment:
belongs_to user, not user_id (like space below)
```suggestion
  belongs_to :user
```